### PR TITLE
initscripts-aero: ds4: Fix enumeration after reboot

### PR DESCRIPTION
**Pull request by zehortigoza**
DS4 power was turned on in BIOS 01.00.13 but this is causing DS4 not
be enumerated after a reboot, so here it is turned off and then on
fixing the enumeration problem.

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
While at it, please translate this to a systemd setvice

**Comment by zehortigoza** [comment]
> While at it, please translate this to a systemd setvice

done

File: recipes-core/initscripts-aero/files/ds4.service [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Configure the power of DS4 camera
```
Comment (lucasdemarchi) [inline recipes-core/initscripts-aero/files/ds4.service:2]: Power on DS4 camera

**Comment by lucasdemarchi** [comment]
This is applied already.

**Pull request closed without merging**